packages/framer-motion/src/render/index.ts

```diff
@@ -515,6 +515,13 @@ export const visualElement =
             },
         }
 
+        element.onUnmount(() => {
+            element.forEachValue((value) => {
+                //console.log(value)
```

Comment:
This can be removed 😄

Reply:
Thanks! Any idea about the TypeScript issues? Not sure what that is about.